python/pecos/classical_interpreters/phir_classical_interpreter.py

```diff
@@ -81,7 +83,7 @@ def init(self, program: str | (dict | QuantumCircuit), foreign_obj: ForeignObjec
             self.program = self.program.to_phir_dict()
 
         # Assume PHIR dict format, validate PHIR
-        if isinstance(self.program, dict):
+        if isinstance(self.program, dict) and self.phir_validate:
             PHIRModel.model_validate(self.program)
```

Comment:
When does this validation kick in? Since we haven't made a new phir release yet, the JSON tests you added should fail validation checks if this line is reachable for them.

Reply:
Ah, never mind, I see you setting them to `false` in the tests.

Reply:
Yeah, I was in the awkward position that the `phir` package wont validate the barriers right now. I'll get rid of turning it off once I can depend a version of `phir` that allows it.